tests/foreman/cli/test_hostcollection.py

```diff
@@ -134,32 +134,36 @@ def test_positive_create_with_limit(module_org):
 
 
 @pytest.mark.tier1
-def test_positive_create_with_unlimited_hosts(module_org):
-    """Create Host Collection with different values of unlimited-hosts
-    parameter
+def test_positive_update_to_unlimited_hosts(module_org):
+    """Create Host Collection with a limit and update it to unlimited hosts
 
     :id: d688fd4a-88eb-484e-9e90-854e0595edd0
 
-    :expectedresults: Host Collection is created and unlimited-hosts
-        parameter is set
+    :expectedresults: Host Collection is created and updated to unlimited hosts
 
-    :CaseImportance: Critical
+    :CaseImportance: High
```

Comment:
Not required to explicitly mention for `High`, it will be taken from module-level docstrings testimony token. Remove the token.